### PR TITLE
Fix inconsistent formatting in README.md

### PR DESCRIPTION
**Pull request by grrttedwards**
In the Scala README.md, there were some inconsistencies in the formatting. The sub-headers under Mac installation were enormous H1-size headers, and the code to run at the bottom was in `inline code form`.

### CODE REVIEW

File: Scala/README.md [hunk 1]
```diff
@@ -11,25 +11,25 @@ Before you start installing Scala on your machine, you must have Java 1.8 or gre
 * Windows: Install [Scala binaries](http://www.scala-lang.org/download/).
 * Linux: Install  
 #### - Ubuntu and other Debian-based distro  
-                   echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list  
-                    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823  
-                    sudo apt-get update  
-                    sudo apt-get install sbt  
+    echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
```
Comment (sbshah97) [inline Scala/README.md]: Replace the entire thing with this
(```)
Check out how to write Markdown for multi-line code!

File: Scala/README.md [hunk 2]
```diff
@@ -11,25 +11,25 @@ Before you start installing Scala on your machine, you must have Java 1.8 or gre
...
-                    sudo apt-get update  
-                    sudo apt-get install sbt  
+    echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list  
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823  
+    sudo apt-get update  
+    sudo apt-get install sbt  
 #### - Red Hat Enterprise Linux and other RPM-based distro  
-                 curl https://bintray.com/sbt/rpm/rpm | sudo tee/etc/yum.repos.d/bintray-sbt-rpm.repo  
-                 sudo yum install sbt  
+    curl https://bintray.com/sbt/rpm/rpm | sudo tee/etc/yum.repos.d/bintray-sbt-rpm.repo
```
Comment (sbshah97) [inline Scala/README.md]: Replace the entire thing with this
(```)
Check out how to write Markdown for multi-line code!

File: Scala/README.md [hunk 3]
```diff
@@ -11,25 +11,25 @@ Before you start installing Scala on your machine, you must have Java 1.8 or gre
...
-                 sudo yum install sbt  
+    curl https://bintray.com/sbt/rpm/rpm | sudo tee/etc/yum.repos.d/bintray-sbt-rpm.repo  
+    sudo yum install sbt  
 * MacOS: Install  
-# [Using Homebrew](https://brew.sh/)
-                    $ brew install sbt  
-# [Using Macports](https://www.macports.org/)
-                    $ port install sbt
+#### - [Using Homebrew](https://brew.sh/)
+    $ brew install sbt
```
Comment (sbshah97) [inline Scala/README.md]: Replace the entire thing with this
(```)
Check out how to write Markdown for multi-line code!

File: Scala/README.md [hunk 4]
```diff
@@ -11,25 +11,25 @@ Before you start installing Scala on your machine, you must have Java 1.8 or gre
...
+    $ port install sbt
  
 
 ### Running Scala code
 Open a command prompt and `cd` to the folder containing your `hello-world.scala` file.
 
 In the command-prompt window, enter the following command to run the program:
 
-`scala hello-world.scala`
+    scala hello-world.scala
```
Comment (sbshah97) [inline Scala/README.md]: Replace the entire thing with this
(```)
Check out how to write Markdown for multi-line code!

File: Scala/README.md [hunk 5]
```diff
@@ -11,25 +11,25 @@ Before you start installing Scala on your machine, you must have Java 1.8 or gre
 * Windows: Install [Scala binaries](http://www.scala-lang.org/download/).
 * Linux: Install  
 #### - Ubuntu and other Debian-based distro  
-                   echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list  
-                    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823  
-                    sudo apt-get update  
-                    sudo apt-get install sbt  
+    echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list  
+    sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823
```
Comment (sbshah97) [inline Scala/README.md]: Replace the entire thing with this
(```)
Check out how to write Markdown for multi-line code!

**Review by sbshah97 — CHANGES_REQUESTED** [review]
Make the required changes on the same branch so that it gets pushed to this PR!

**Comment by grrttedwards** [comment]
Updated markdown to use "multi-line" markdown for the shell commands.

**Comment by grrttedwards** [comment]
If the merge could wait until tomorrow, that would be cool :) 
I didn't know Hacktoberfest was ONLY Oct 1-31 :( My other PR #125 was merged today.

**Comment by sbshah97** [comment]
Not a problem! We'll wait. Just make sure you've referenced the Issue!

**Comment by grrttedwards** [comment]
Fixes issue #133

**Review by sbshah97 — APPROVED** [review]

**Pull request merged**